src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,26 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo:FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <>
+      <article className={`TodoInfo ${completed ? 'TodoInfo--completed' : ''}`}>
+        <h2 className="TodoInfo__title">{title}</h2>
+
+        {user && <UserInfo user={user} />}
+      </article>
+    </>
```

Comment:
Why did you use react fragment here?
```suggestion
      <article className={`TodoInfo ${completed ? 'TodoInfo--completed' : ''}`}>
        <h2 className="TodoInfo__title">{title}</h2>

        {user && <UserInfo user={user} />}
      </article>
```